webui/src/lib/components/repository/changes.jsx

```diff
@@ -126,31 +128,25 @@ export const TreeEntryPaginator = ({ path, setAfterUpdated, nextPage, depth=0, l
     );
 };
 
-function useTreeItemType(entry, repo, leftDiffRefID, rightDiffRefID) {
-    const [treeItemType, setTreeItemType] = useState({type: null, loading: true});
-
+async function useTreeItemType(entry, repo, leftDiffRefID, rightDiffRefID, isDeltaEnabled) {
```

Comment:
Why does this PR need to change this function?  Is this a change that was intended for another PR?